odl/operator/operator.py

```diff
@@ -707,6 +707,54 @@ def __call__(self, x, out=None, **kwargs):
                     raise_from(new_exc, err)
         return out
 
+    def norm(self, estimate=False, **kwargs):
```

Comment:
Quite skeptical to a global way of doing this. The main reason is that `Operator`s are immutable through-out ODL, which means that if I am given an operator, I know it will always return the same thing at all times, and I don't have to be afraid of it changing values due to some change. While we really only depend on this property for `__call__`, it is health to try to keep it that way (simply because code becomes so much easier to debug).

Even so, this framework allows you to manually set the norm in sub-classes, its just that we don't provide a global way of doing it.

Is there any specific use-case where you would want to set it manually? Perhaps that would give some more insight into how to design this.